CANdy/test/README.md

```diff
@@ -0,0 +1,17 @@
+# CANdy-Test Network Genesis Files - Fichier Genesis du registre du réseau de la couche test CANdy
+
+The CANdy-Test network is a public based block-chain network - Le réseau de la couche CANdy-Test est public et il est basé sur une chaîne de blocs. 
+
+[Hyperledger Indy-Node/noeud](https://github.com/hyperledger/indy-node) 
+
+The source files and configurations used to establish this network can be found in the repository - Les fichiers sources et les configurations utilisés pour établir ce réseau sont disponibles dans le dépôt suivant :
+
+[CQEN-QDCE/Candy](https://github.com/CQEN-QDCE/Candy)
+
+## [pool_transactions_genesis](./pool_transactions_genesis)
+
+This file contains the set of genesis nodes for the CANdy-Test network.  This is the genesis file to use when establishing a client/agent connection to the network. - Ce fichier contient l'ensemble des nœuds Genesis du registre du réseau de la couche test CANdy. Il s'agit du fichier Genesis à utiliser lors de l'établissement d'une connexion client/agent au réseau :
+
+## [domain_transactions_genesis](./domain_transactions_genesis)
+
+This file contains the set of genesis Trustees and Stewards for the CANdy-Test network. - Ce fichier contient l'ensemble du fichier Genesis Trustees et Stewards du registre du réseau de la couche test CANdy.
```

Comment:
"Ce fichier contient l'ensemble du fichier Genesis Trustees et Stewards du registre du réseau de la couche test CANdy." should be written "Ce fichier contient les informations des Administrateurs de confiance et des Intendants d'origine pour la couche Test du réseau CANdy." in French.